src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -425,16 +468,16 @@ export default class OrgPerson extends Mixins(CommonMixin) {
       person.officer.id = uuidv4()
     }
     person.mailingAddress = { ...this.inProgressMailingAddress }
-    if (this.isDirector) {
+    if (this.isDirector || this.isProprietor || this.isPartner) {
```

Comment:
What about the completing party's delivery address (firms only)?

Reply:
Completing Party isn't a factor in this component (for firms) afaik
Completing party is manually added or pulled from the account for this filing in another component on the review page (not completed).

Reply:
Ask UI/UX for confirmation. In Create UI, CP gets a delivery address for firms.